AEPCore/Sources/eventhub/EventHub.swift

```diff
@@ -30,6 +30,7 @@ final class EventHub {
     private let eventQueue = OperationOrderer<Event>("EventHub")
     private var preprocessors = ThreadSafeArray<EventPreprocessor>(identifier: "com.adobe.eventHub.preprocessors.queue")
     private var started = false // true if the `EventHub` is started, false otherwise. Should only be accessed from within the `eventHubQueue`
+    private var eventHistory = EventHistory()
```

Comment:
How are we going to pass the `EventHistory` instance to `RulesEngine`s? I guess we need to add a public getter in `ExtensionContainer`/`ExtensionRuntime`, which will be passed to the RulesEngine instance here (https://github.com/adobe/aepsdk-core-ios/blob/14dbeef70cb1375ea86667db5571e74522ccc23f/AEPCore/Sources/rules/LaunchRulesEngine.swift#L51), right?
If so, can we also make those changes in this PR?